sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.2.0-beta.2 (Unreleased)
+## 1.2.0-beta.2 (2020-10-05)
 
```

Comment:
David and I were talking about what could go to the release highlights from these changes. We've agreed on the first three items so far.

So, this would be in the release highlights:

- `DefaultAzureCredential` now by default shows the Device Code message on the console. This can still be overwritten with a custom behavior by specifying a function as the third parameter, `userPromptCallback`.
- Added support for multiple clouds on `VisualStudioCodeCredential`. Fixes customer issue [11452](https://github.com/Azure/azure-sdk-for-js/issues/11452).
- `ManagedIdentityCredential` has been aligned with other languages, now treating expected errors properly. This fixes customer issue [11451](https://github.com/Azure/azure-sdk-for-js/issues/11451).

Thoughts?

Reply:
Added  ADFS authority Host support to the node credentials.